### DISCUSSION TITLE
[FEATURE] Why does `Documentation Coverage` not account for types/type aliases?

### DISCUSSION THREAD

**Discussion by edumserrano** [compodoc/compodoc#1516] [Q&A] [answered]
When looking at the `Documentation Coverage` for my library it shows:

- function
- variable
- injectable
- class
- interface
- directive

But I have many types under the `Miscellaneous -> Type Aliases` section that aren't checked for documentation coverage. 

Are types not supported for `Documentation Coverage` or am I doing something wrong?

**Comment by vogloblinsky** [comment, marked as answer]
Hi,
Good remark, i fix that.
Thanks

**Reply by edumserrano** [reply]
Hi @vogloblinsky,

I've seen you merged a PR for this, any idea when there will be a release with this change?

**Reply by edumserrano** [reply]
I can see this was recently released with [version 1.1.26](https://github.com/compodoc/compodoc/releases/tag/1.1.26). Thank you 🥇

**Comment by edumserrano** [comment]
Since this discussion has been moved to a PR #1517, I'll mark this as closed.
